src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { PostInfo } from '../PostInfo';
+import { PostArr } from '../../type/post';
+import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+export const PostList: React.FC<PostArr> = ({ posts }) => (
+  <div className="PostList">
```

Comment:
use semantic tag - `section` or even better `ul` (and then set a proper tag for children, i.e. `article` or `li` inside `PostInfo` component)